### PR TITLE
[IIDM v1.3] Rename to setpoint (#162)

### PR DESCRIPTION
**Pull request by sebalaig**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#162 

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/iidm/converter/xml/StaticVarCompensatorXml.cpp [hunk 1]
```diff
@@ -37,8 +37,14 @@ const char* StaticVarCompensatorXml::getRootElementName() const {
...
-    double voltageSetpoint = context.getReader().getOptionalAttributeValue(VOLTAGE_SET_POINT, stdcxx::nan());
-    double reactivePowerSetpoint = context.getReader().getOptionalAttributeValue(REACTIVE_POWER_SET_POINT, stdcxx::nan());
+    std::string voltageSetpointName = VOLTAGE_SETPOINT;
+    std::string reactivePowerSetpointName = REACTIVE_POWER_SETPOINT;
+    IidmXmlUtil::runUntilMaximumVersion(IidmXmlVersion::V1_2(), context.getVersion(), [&voltageSetpointName, &reactivePowerSetpointName]() {
+        voltageSetpointName = VOLTAGE_SET_POINT;
+        reactivePowerSetpointName = REACTIVE_POWER_SET_POINT;
+    });
+    double voltageSetpoint = context.getReader().getOptionalAttributeValue(voltageSetpointName, stdcxx::nan());
+    double reactivePowerSetpoint = context.getReader().getOptionalAttributeValue(reactivePowerSetpointName, stdcxx::nan());
```
Comment (mathbagu) [inline src/iidm/converter/xml/StaticVarCompensatorXml.cpp:47]: honestly, I'm not sure this implementation is really readable even if it matches to what is done in the Java version. I would prefer to initialize each voltageSetpointName and reactivePowerSetpointName using a ternary operator or a classic if/else block

What do you think?

File: src/iidm/converter/xml/StaticVarCompensatorXml.cpp [hunk 2]
```diff
@@ -69,8 +75,14 @@ void StaticVarCompensatorXml::readSubElements(StaticVarCompensator& svc, Network
...
-    context.getWriter().writeAttribute(VOLTAGE_SET_POINT, svc.getVoltageSetpoint());
-    context.getWriter().writeAttribute(REACTIVE_POWER_SET_POINT, svc.getReactivePowerSetpoint());
+    std::string voltageSetpointName = VOLTAGE_SETPOINT;
+    std::string reactivePowerSetpointName = REACTIVE_POWER_SETPOINT;
+    IidmXmlUtil::runUntilMaximumVersion(IidmXmlVersion::V1_2(), context.getVersion(), [&voltageSetpointName, &reactivePowerSetpointName]() {
+        voltageSetpointName = VOLTAGE_SET_POINT;
+        reactivePowerSetpointName = REACTIVE_POWER_SET_POINT;
+    });
+    context.getWriter().writeAttribute(voltageSetpointName, svc.getVoltageSetpoint());
+    context.getWriter().writeAttribute(reactivePowerSetpointName, svc.getReactivePowerSetpoint());
```
Comment (mathbagu) [inline src/iidm/converter/xml/StaticVarCompensatorXml.cpp:85]: same remark here

File: src/iidm/converter/xml/StaticVarCompensatorXml.cpp (hunk 1 above)
Comment (sebalaig) [reply src/iidm/converter/xml/StaticVarCompensatorXml.cpp:47]: Fully agree, I would prefer using a ternary operator.

Comment (sebalaig) [reply src/iidm/converter/xml/StaticVarCompensatorXml.cpp]: Fixed in d35a81e890dba299cbc45f2b41fcd95364b5979b

**Review by mathbagu — APPROVED** [review]

**Pull request merged**